src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -30,15 +30,21 @@ class LineChart extends ChartComponent {
     private hasStackedButton: boolean = false;
     private stackedButton: any = null;
     private gridButton: any = null;
+    private scooterButton: any = null;
 
     public x: any;
+    private yMap: any;
+    private colorMap: any;
+    private activeScooter: any;
+    private scooterGUIDMap: any = {};
```

Comment:
scooterGuidMap?  prefer non-consecutive capital letters for stricter camelcasing